deeplite/tflite_profiler/tflite_inference.py

```diff
@@ -0,0 +1,53 @@
+import numpy as np
+import tensorflow as tf
+tf.compat.v1.logging.set_verbosity(tf.compat.v1.logging.ERROR)
+try:
+    tf.compat.v1.enable_eager_execution()
+except:
+    pass
+
+from deeplite.profiler.evaluate import EvaluationFunction
+from deeplite.profiler.utils import Device
+
+
+def top_k_accuracy(results, gt_labels, k, dtype="uint8"):
+    accuracy = (
+        np.sum(np.argsort(results, axis=1)[:, -k:] == gt_labels.reshape(-1, 1)) /
+        gt_labels.size)
+    return accuracy * 100.0
+
+
+class _GetAccuracy(EvaluationFunction):
+    def __call__(self, model, data_loader, device=Device.CPU, transform=None, k=1):
+
+          """Evaluates tensorflow lite classification model with the given dataset."""
+          interpreter = tf.lite.Interpreter(model_content=model)
+          interpreter.allocate_tensors()
+          input_idx = interpreter.get_input_details()[0]['index']
+          output_idx = interpreter.get_output_details()[0]['index']
+          input_details = interpreter.get_input_details()[0]
+          dtype = input_details['dtype']
+          results = []
+          gt_labels = []
+          for x, y in data_loader:
+            if (len(x.shape) == 3):
+                x = x[None]
+            if transform:
+                x, y = transform(x, y)
```

Comment:
add test coverage

Reply:
The transform is an input param, we don't need it here

Reply:
added coverage to transform